api/turing/validation/validator.go

```diff
@@ -148,6 +149,53 @@ func checkDefaultTrafficRule(
 	}
 }
 
+func validateDefaultRouteTrafficRules(
+	sl validator.StructLevel,
+	fieldName string,
+	defaultTrafficRules models.TrafficRules,
```

Comment:
Super nit: Why `defaultTrafficRules` here? Shall we just call it `trafficRules`  ? 😅

Reply:
Oh yes thats a typo, updated! 🙏🏻